deny.toml

```diff
@@ -101,6 +101,14 @@ exceptions = [
   # list
   { allow = ["Unicode-DFS-2016"], crate = "unicode-ident" },
   { allow = ["CC0-1.0"], crate = "tiny-keccak" },
+  { allow = ["MPL-2.0"], crate = "colored" },
+  { allow = ["Zlib"], crate = "foldhash" },
+  { allow = ["MPL-2.0"], crate = "indent" },
+  { allow = ["BSD-3-Clause"], crate = "subtle" },
+  { allow = ["MIT"], crate = "unescaper" },
+  { allow = ["Zlib"], crate = "const_format" },
+  { allow = ["Zlib"], crate = "const_format_proc_macros" },
+  { allow = ["CC0-1.0"], crate = "constant_time_eq" },
```

Comment:
These are more crates than you list in the `CREDITS.md` file. Are the ones not listed okay without any special action?

Reply:
Yes only MPL-2 and BSD-3 libraries need to be listed for the time being.

Reply:
Okay.